link2aws.js

```diff
@@ -993,6 +993,7 @@ class ARN {
             "states": { // AWS Step Functions
                 "activity": null,
                 "execution": null,
```

Comment:
Minor issue: the null line should've been removed so that there are no duplicates. I'll update this and merge it.